Plugins/Alpakit/Source/Alpakit/Private/AlpakitModEntryList.cpp

```diff
@@ -108,3 +120,48 @@ void SAlpakitModEntryList::SetShowEngine(bool bInShowEngine) {
 	bShowEngine = bInShowEngine;
 	LoadMods();
 }
+
+FReply SAlpakitModEntryList::PackageAllMods() {
+	TSharedPtr<SAlpakitModEntry> first;
+	TArray<TSharedPtr<SAlpakitModEntry>> nextEntries;
+
+	UE_LOG(LogTemp, Display, TEXT("Alpakit All!"))
+
+	for (auto mod : FilteredMods) {
+		UE_LOG(LogTemp, Display, TEXT("Collecting Plugin %s!"), *mod->GetName())
+
+		auto tableRow = ModList->WidgetFromItem(mod);
+		if (!tableRow.IsValid()) {
+			UE_LOG(LogTemp, Display, TEXT("TableRow not found!"))
+
+			continue;
+		}
+
+		auto modEntry = StaticCastSharedPtr<SAlpakitModEntry>(tableRow->GetContent());
+		if (!modEntry.IsValid()) {
+			UE_LOG(LogTemp, Display, TEXT("TableRow content is not valid!"))
+
+			continue;
+		}
+
+		if(!modEntry->IsSelected()) {
+			UE_LOG(LogTemp, Display, TEXT("Plugin is not selected: %s"), * mod->GetName());
```

Comment:
Is that an abnormal condition? do you really need to log it?

Reply:
It was made for debugging purposely. Left overs